common/src/main/resources/assets/figura/lang/en_us.json

```
@@ -1130,33 +1131,33 @@
     "figura.docs.model_part": "Represents a node in the model tree, basically a group/cube/mesh in Blockbench\nEach bbmodel file is itself a ModelPart, and all of your models are contained in a global ModelPart called \"models\"",
     "figura.docs.model_part.pre_render": "Function to run before this part starts being rendered",
     "figura.docs.model_part.mid_render": "Function to run during the middle of this part's rendering, after its matrices are calculated",
-    "figura.docs.model_part.post_render": "Function to run after this part and its children is rendered",
+    "figura.docs.model_part.post_render": "Function to run after this part and its children are rendered",
     "figura.docs.model_part.set_pre_render": "Sets a function to run before this part starts being rendered",
     "figura.docs.model_part.set_mid_render": "Sets a function to run during the middle of this part's rendering, after its matrices are calculated",
-    "figura.docs.model_part.set_post_render": "Sets a function to run after this part and its children is rendered",
+    "figura.docs.model_part.set_post_render": "Sets a function to run after this part and its children are rendered",
     "figura.docs.model_part.get_name": "The name of this model part",
     "figura.docs.model_part.get_parent": "Gets the parent part of this part\nIf this part has no parent, returns nil",
     "figura.docs.model_part.get_children": "Gets the children of this part, stored in a table",
     "figura.docs.model_part.is_child_of": "Checks if this part is a child of the given part",
-    "figura.docs.model_part.get_pos": "Gets the position of the model part, as an offset from its position in blockbench\nOnly changes from {0,0,0} when you call setPos()",
-    "figura.docs.model_part.set_pos": "Sets the position offset for this part from its blockbench position\nNil values for position are assumed to be 0",
+    "figura.docs.model_part.get_pos": "Gets the position of the model part, as an offset from its position in BlockBench\nOnly changes from {0,0,0} when you call setPos()",
+    "figura.docs.model_part.set_pos": "Sets the position offset for this part from its BlockBench position\nNil values for position are assumed to be 0",
```

The B in bench is not capitalized. [See Blockbench's site.](https://www.blockbench.net/)
```suggestion
    "figura.docs.model_part.get_pos": "Gets the position of the model part, as an offset from its position in Blockbench\nOnly changes from {0,0,0} when you call setPos()",
    "figura.docs.model_part.set_pos": "Sets the position offset for this part from its Blockbench position\nNil values for position are assumed to be 0",
```

---

~~I'm not sure what you mean by this... Every instance of the word "Blockbench" on their site is capitalized as far as I can tell.~~

Edit: NVM I misread. You are correct.